deploy/alerts/simple-comparison.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: fixed-compares
+    role: alert-rules
+  name: prometheus-alarms-part-two
+  namespace: service-telemetry
+spec:
+  groups:
+    - name: ./example.rules
```

Comment:
Does this get merged automatically when you use the same file name? See suggested file name from earlier.

Reply:
it doesn't get merged. I haven't found any docs or reason why this tag is there at all.